src/autorest-core/package.json

```diff
@@ -44,7 +44,7 @@
     "jsonpath": "^0.2.11",
     "linq-es2015": "^2.4.13",
     "oav": "0.4.1",
-    "azure-openapi-validator": "git+https://github.com/olydis/azure-openapi-validator.git#b45423f4560585e64f574bbffbb5cdf894cfbd53",
+    "azure-openapi-linter": "^0.1.4",
```

Comment:
Let's wait until we release `azure-openapi-validator` version and update this so you don't have to do back & forth.